api/app/signals/apps/signals/migrations/0097_fix_parent_prullenbak-vol.py

```diff
@@ -0,0 +1,89 @@
+"""
+Fixing the parent of category "prullenbak-kapot" from "schoon" to "wegen-verkeer-straatmeubilair"
+"""
+
+from django.db import migrations
+
+NEW_CATEGORIES = {
+    'wegen-verkeer-straatmeubilair': {
+        'prullenbakkapot': {  # noqa SIG-2242 Verplaats: Prullenbak is kapot (was: Afval) naar Wegen, Verkeer en Straatmeubilair
+            'name': 'Prullenbak is kapot',
+            'handling': 'A3DEC',
+            'slo': '3W',
+            'departments': [
+                'STW',
+            ]
+        },
+    },
+}
+
+ML_TRANSLATION = (
+    ('prullenbak-is-kapot', 'prullenbakkapot'),  # noqa SIG-2242 Verplaats: Prullenbak is kapot (was: Afval) naar Wegen, Verkeer en Straatmeubilair
+    ('prullenbak-kapot', 'prullenbakkapot'),  # noqa SIG-2242 Verplaats: Prullenbak is kapot (was: Afval) naar Wegen, Verkeer en Straatmeubilair
```

Comment:
Check that a double translation is possible (prior migration added `'prullenbak-is-kapot'` -> `'prullenbak-kapot'`).

Does the new translation `'prullenbak-is-kapot'` -> `'prullenbakkapot'` overwrite the old one? If not we may get weird behavior in the ML proxy. Looking at model class `CategoryTranslation` there is no uniqueness constraint on the name of the old category --- please check this.

And related `'prullenbak-kapot'` -> `'prullenbakkapot'` may not be necessary.

Reply:
as discussed via Slack, this should be fine